byterun/major_gc.c

```diff
@@ -231,7 +231,7 @@ static void init_sweep_phase(void)
 }
 
 /* auxillary function of mark_slice */
-static inline value* mark_slice_darken(value *gray_vals_ptr, value v, int i,
+static inline value* mark_slice_darken(value *gray_vals_ptr, value v, intnat i,
```

Comment:
`i` is a field offset, so it should probably be `mlsize_t` (and that's what gets passed for this parameter in `mark_slice`).

Reply:
OK, commit fd3fd43 uses `mlsize_t` as suggested.